projects/petri-svg/src/lib/canvas/svg-elements/svg-objects/canvas-element.ts

```diff
@@ -4,6 +4,7 @@ import {Container} from './container';
 export abstract class CanvasElement extends Container implements Activable {
 
     private _isSelected = false;
+    protected changable = true;
```

Comment:
Is this locking needed in petri-svg lib? Can't it be done one layer above in petriflow-canvas or custom project?

Reply:
it is needed, because css classes and properties are changing on every hover on object, and its impossible to highlight permanently some svg objects, because highlight is deleted after hover

Reply:
it works on petriflow-svg demo, see https://netgrif.github.io/petriflow.svg/app
select tool can block this behavior, if you select multiple elements they stay highlighted
lets discuss this, please contact me on Slack or at office